root/instrument/List.js

```diff
@@ -0,0 +1,47 @@
+const React = require('react');
+const Layout = require('../layout');
+const EntityLink = require('../static/scripts/common/components/EntityLink');
+const {l, lp} = require('../static/scripts/common/i18n');
+
+const instrumentDescription = (description) => ({ __html: " &mdash; " + description });
+const Instrument = (props) => {
+  const instrument = props.instrument;
+  return (
+    <li>
+      <EntityLink entity={instrument} />
+      { instrument.description ? <span dangerouslySetInnerHTML={instrumentDescription(instrument.description)} /> : null }
```

Comment:
I'd remove the `instrumentDescription` function and just inline it:

```
={instrumentDescription(instrument.description)}
vs.
={{__html: ' — ' + instrument.description}}
```

You can write the em-dash directly instead of using the `&mdash;` character entity reference.

Also remove the spaces `{ around this }` so it's `{like this}` to match our existing style.

Reply:
How about this, so the HTML makes a bit more sense (import `Frag` from root/components/Frag.js):
```JSX
{instrument.description
  ? <Frag>
      {' — '}
      <span className="description"
            dangerouslySetInnerHTML={{__html: instrument.description}} />
    </Frag>
  : null}
```